modules/phase_field/src/materials/ComputePolycrystalElasticityTensor.C

```diff
@@ -48,9 +48,6 @@ ComputePolycrystalElasticityTensor::computeQpElasticityTensor()
   const std::vector<std::pair<unsigned int, unsigned int> > & active_ops = _grain_tracker.getElementalValues(_current_elem->id());
```

Comment:
For cases where we have to loop over order parameters isn't it better to get the active ones only?